module_intros/density.LocalDensity.ipynb

```diff
@@ -0,0 +1,257 @@
+{
```

Comment:
<p>The density *decreases* as we move up and to the right.</p><p> </p><p>Also, remove the references to reference points. Since the API is now standardized, I would reword this paragraph to indicate that "we can take advantage of the <code>query_points</code> API to find the density of our points around the cluster centers" or something similar, i.e. don't emphasize the API since it's not special in any way.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/freud-examples/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>